rust/cubesql/cubesql/src/sql/statement.rs

```diff
@@ -657,6 +656,39 @@ impl<'a> Visitor<'a> for UdfWildcardArgReplacer {
     }
 }
 
+#[derive(Debug)]
+pub struct SensitiveDataSanitizer {}
+
+impl SensitiveDataSanitizer {
+    pub fn new() -> Self {
+        Self {}
+    }
+
+    pub fn replace(mut self, stmt: &ast::Statement) -> ast::Statement {
+        let mut result = stmt.clone();
+
+        self.visit_statement(&mut result);
+
+        result
+    }
+}
+
+impl<'ast> Visitor<'ast> for SensitiveDataSanitizer {
+    fn visit_value(&mut self, val: &mut ast::Value) {
+        match val {
+            ast::Value::SingleQuotedString(str)
+            | ast::Value::DoubleQuotedString(str)
+            | ast::Value::NationalStringLiteral(str) => {
+                if vec!["f", "false", "t", "true"].contains(&str.as_str()) {
+                    return;
+                }
+                *str = "[REPLACED]".to_string();
```

Comment:
Minor: `[REDACTED]` sounds like being used just for that everywhere, should we change the string?